spec/tiny_server.rb

```diff
@@ -78,12 +80,13 @@ def url
     end
 
     def block_until_started
-      200.times do
-        if started? && !@server.nil?
+      600.times do
+        if !@http_server.nil? && started?
```

Comment:
I switched it because no need to make HTTP calls in `started?` if there is no server. Not a critical reordering.